templates/gitlab-ci-template-cloud-run.yml

```diff
@@ -71,12 +73,12 @@ workflow:
       else
         export SERVICE_ACCOUNT_OPT=""
       fi
-    # If the DEPLOYMENT_SUFFIX variable is not defined, the FULL_DEPLOYMENT_NAME is equal to DEPLOYMENT_NAME otherwise the suffix will be appended.
+    # If the DEPLOYMENT_SUFFIX variable is not defined, the DEPLOYMENT_FULL_NAME is equal to DEPLOYMENT_NAME otherwise the suffix will be appended.
     - |
       if [ -n "${DEPLOYMENT_SUFFIX}" ]; then
-        export FULL_DEPLOYMENT_NAME="${DEPLOYMENT_NAME}-${DEPLOYMENT_SUFFIX}"
```

Comment:
i was thinking if it's really needed to have a DEPLOYMENT_SUFFIX variable.

What if the project has to define the project name and leave the project to decide what to append or not ? It's up to you to use something like `foobar-${CI_COMMIT_REF_SLUG}`.

Maybe we can just provide just an example.

Reply:
We can safely remove the `DEPLOYMENT_SUFFIX`. It was intended to help with the deployment of multiple services, for example multiple deployment with different country code. In the end the developers can overwrite the `DEPLOYMENT_NAME` using what they need.

Reply:
Fixed!